plotly/offline/offline.py

```diff
@@ -161,7 +161,8 @@ def _get_jconfig(config):
 
     plotly_platform_url = plotly.tools.get_config_plotly_server_url()
 
-    clean_config['plotlyServerURL'] = plotly_platform_url
+    if not clean_config.get('plotlyServerURL'):
```

Comment:
I think we want `clean_config.get('plotlyServerURL', None)` in case config doesn't include `'plotlyServerURL'` at all.